pom.xml

```diff
@@ -695,7 +695,7 @@
                             <dependency>
                                 <groupId>org.slf4j</groupId>
                                 <artifactId>slf4j-api</artifactId>
-                                <version>1.7.36</version>
+                                <version>2.0.13</version>
```

Comment:
PR should be closed, this change will break the DataNucleus Maven plugin, as per:

https://github.com/stevespringett/Alpine/blob/e747ed50be81f9fc955a44bf8f7786a26973e99f/pom.xml#L691-L694